{
  "uid" : "fe34bcbd6bd4a77a",
  "name" : "登录反向用例测试",
  "fullName" : "cases.test_login.TestLogin#test_login_failed",
  "historyId" : "28fbee3f7849e5c692210a9f12e7fe96",
  "time" : {
    "start" : 1615354526979,
    "stop" : 1615354528614,
    "duration" : 1635
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: type object 'BassPage' has no attribute 'checkbox_agree'",
  "statusTrace" : "self = <cases.test_login.TestLogin object at 0x0000013DED42DEF0>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"74106055-0537-45ec-b9d5-48f170b106f8\")>, <pages.page_action.PageAction object at 0x0000013DEC010E10>)\ndata = {'error_msg': '登录验证失败，用户不存在！', 'name': '登录-反向用例-用户名错误', 'password': '654321', 'username': '财友'}\n\n    @allure.story('钓鱼人APP自动化测试-登录模块-反向用例测试')\n    @allure.title('登录反向用例测试')\n    @pytest.mark.parametrize('data',failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n>       BassPage.checkbox_agree()\nE       AttributeError: type object 'BassPage' has no attribute 'checkbox_agree'\n\ncases\\test_login.py:38: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "prepare_to_login",
    "time" : {
      "start" : 1615354513461,
      "stop" : 1615354521534,
      "duration" : 8073
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "handler",
    "time" : {
      "start" : 1615354485161,
      "stop" : 1615354513384,
      "duration" : 28223
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "AttributeError: type object 'BassPage' has no attribute 'checkbox_agree'",
    "statusTrace" : "self = <cases.test_login.TestLogin object at 0x0000013DED42DEF0>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"74106055-0537-45ec-b9d5-48f170b106f8\")>, <pages.page_action.PageAction object at 0x0000013DEC010E10>)\ndata = {'error_msg': '登录验证失败，用户不存在！', 'name': '登录-反向用例-用户名错误', 'password': '654321', 'username': '财友'}\n\n    @allure.story('钓鱼人APP自动化测试-登录模块-反向用例测试')\n    @allure.title('登录反向用例测试')\n    @pytest.mark.parametrize('data',failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n>       BassPage.checkbox_agree()\nE       AttributeError: type object 'BassPage' has no attribute 'checkbox_agree'\n\ncases\\test_login.py:38: AttributeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "ce9344ce7951fa35",
      "name" : "log",
      "source" : "ce9344ce7951fa35.txt",
      "type" : "text/plain",
      "size" : 3879
    } ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 0,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "handler::0",
    "time" : {
      "start" : 1615354535067,
      "stop" : 1615354536393,
      "duration" : 1326
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "钓鱼人APP自动化测试-登录模块"
  }, {
    "name" : "epic",
    "value" : "钓鱼人APP自动化测试"
  }, {
    "name" : "story",
    "value" : "钓鱼人APP自动化测试-登录模块-反向用例测试"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('prepare_to_login')"
  }, {
    "name" : "parentSuite",
    "value" : "cases"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "LAPTOP-LSM6I36P"
  }, {
    "name" : "thread",
    "value" : "12444-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "cases.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "data",
    "value" : "{'name': '登录-反向用例-用户名错误', 'username': '财友', 'password': '654321', 'error_msg': '登录验证失败，用户不存在！'}"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.usefixtures('prepare_to_login')" ]
  },
  "source" : "fe34bcbd6bd4a77a.json",
  "parameterValues" : [ "{'name': '登录-反向用例-用户名错误', 'username': '财友', 'password': '654321', 'error_msg': '登录验证失败，用户不存在！'}" ]
}